core/src/tile/tileManager.cpp

```diff
@@ -173,6 +173,8 @@ TileManager::~TileManager() {
 
 void TileManager::setTileSources(const std::vector<std::shared_ptr<TileSource>>& _sources) {
 
+    LOG("setTileSources %d", _sources.size());
```

Comment:
Don't need to log this all the time

Reply:
yeah - this was for tracking down the 'clientTileSource add' bug on ios